group_vars/mgrs.yml.sample

```diff
@@ -24,6 +24,7 @@ dummy:
 # Ceph mgr modules to enable, to view the list of available mpdules see: http://docs.ceph.com/docs/CEPH_VERSION/mgr/
 # and replace CEPH_VERSION with your current Ceph version, e,g: 'mimic'
 #ceph_mgr_modules: []
+##ceph_mgr_prometheus_plugin_port: 9283
```

Comment:
not sure to understand why there's two ## characters here.

Did you run the `generate_group_vars_sample.sh` script ?

Reply:
No, I've just read #contribution-guidelines =) I'll fix it